control/content/index.html

```diff
@@ -19,7 +19,25 @@
   <script src="assets/js/moment.min.js"></script>
   <script src="../../../../scripts/jquery/jquery-ui.min.js"></script>
   <script src="../../../../scripts/angular/angular-ui-sortable.js"></script>
-  <script src="https://maps.googleapis.com/maps/api/js?v=3.exp&key=AIzaSyBdhPtd93zNB7sXm_sjseo9gZgtwgxRUWM&libraries=places"></script>
+  <script>
+    function setGoogleMapsScript(key) {
+        const docHead = document.getElementsByTagName('head');
+        let scriptEl = document.createElement('script');
+        scriptEl.id = "googleScript";
+        scriptEl.src = "https://maps.googleapis.com/maps/api/js?v=3.exp&libraries=places&key=" + key;
+        docHead[0].appendChild(scriptEl);
+    };
+    
+    var isMapReady = false;
+    
+    function mapReady(){
+      isMapReady = true;
+    }
+
+    buildfire.getContext((error, context) => {
+      setGoogleMapsScript(context.apiKeys.googleMapKey);
+    });
+  </script>
```

Comment:
Make a script so we could reuse it.

Reply:
This potentially could be thousands of apps. This particular plugin may be less. But we need to be careful about doing this for any plugin